moztelemetry/filter_service.py

```diff
@@ -251,14 +252,7 @@ def publish(prefix):
                     if (from_submission_date is None or dims["submissionDate"] >= from_submission_date) and \
                        (to_submission_date is None or dims["submissionDate"] <= to_submission_date) and \
                        dims["submissionDate"][:-2] in sdb:
-                        attributes = {"appName": dims.get("appName"),
-                                      "sourceName": dims.get("sourceName"),
-                                      "docType": dims.get("docType"),
-                                      "submissionDate": dims.get("submissionDate"),
-                                      "appVersion": dims.get("appVersion"),
-                                      "sourceVersion": dims.get("sourceVersion"),
-                                      "appUpdateChannel": dims.get("appUpdateChannel"),
-                                      "appBuildID": dims.get("appBuildId")}
+                        attributes = dims
```

Comment:
Maybe this variable is unnecessary now, since it is just `dims`. At any rate, I didn't find the definition of `schema.get_dimension_map` in this repository but I assume its results are of the same format as `attributes`.